dvc/stage/loader.py

```diff
@@ -91,10 +103,14 @@ def __contains__(self, name):
 
 
 class SingleStageLoader(collections.abc.Mapping):
-    def __init__(self, dvcfile, stage_data, stage_text=None):
+    def __init__(self, dvcfile, stage_data, stage_text=None, tag=None):
         self.dvcfile = dvcfile
         self.stage_data = stage_data or {}
         self.stage_text = stage_text
+        self.tag = tag
+
+    def filter(self, item=None):
+        return self
```

Comment:
```suggestion
        assert item is None
        return self
```
?

Reply:
It's a bit permissible for users to do `stage.dvc:stage_name`.